apps/wallet-ui/src/lib/revalidate.ts

```diff
@@ -0,0 +1,7 @@
+"use server";
+
+import { revalidatePath, revalidateTag } from "next/cache";
+
+export async function revalidate(tag: string) {
+  revalidateTag(tag);
```

Comment:
what are we revalidating?

Reply:
Anything, Next just won't let me call revalidateTag directly in a client component